README.md

```diff
@@ -8,35 +8,35 @@ The `master` branch only contains the `SimpleSwap` contract for now. Everything
 
 ## Tests
 
-This is a regular truffle project. You can either use `truffle test` (if installed) or `npm test` which will use a locally installed truffle.
+This is a hardhat project using the truffle plugin (for tests as this used to be truffle-based). You can run tests using `yarn test`.
 
 ```sh
-npm install
-npm test
+yarn
+yarn test
 ```
 
-To also generate coverage information use `npm run coverage` instead.
+To also generate coverage information use `yarn hardhat coverage` instead.
 
 ## Linting
 
-This repo currently uses `solhint` as linter. It can be called through npm:
+This repo currently uses `solhint` as linter. It can be called through yarn:
 ```sh
-npm run solhint
+yarn lint
```

Comment:
I get a bunch of error messages here. Shouldn't we do something about these, especially taking into account upcoming audit
![image](https://user-images.githubusercontent.com/33000441/111455149-ba9f5800-8715-11eb-8188-7d2f62e6c59d.png)

Reply:
let's address any contract changes in the proxy pr. this one is only about the toolchain.